lib/genevalidator/validation_alignment.rb

```diff
@@ -124,32 +124,30 @@ def initialize(prediction, hits)
     # Output:
     # +AlignmentValidationOutput+ object
     def run(n = 10)
-      n = opt[:min_blast_hits] < 10 ? 10 : opt[:min_blast_hits]
+      n = opt[:min_blast_hits] < 10 ? n : opt[:min_blast_hits]
```

Comment:
This looks like a potentially breaking change. Previously `n` couldn't be lower than 10 (or higher than 50 - L128). With this change, the value of `n` can be set to lower than 10 by the caller.

Reply:
no I don't think this is a breaking change - the variable n in set to 10 in the line above... (so it still cannot be higher than 10...

(This was a change suggested by rubocop)